handlers.go

```diff
@@ -3689,16 +3671,27 @@ func (s *Server) handleCreateContent(c echo.Context, u *User) error {
 		return err
 	}
 
-	for _, col := range collections {
+	if req.Collection != "" {
+		var path *string
+		if req.CollectionPath != "" {
+			sp, err := sanitizePath(req.CollectionPath)
+			if err != nil {
+				return err
+			}
+
+			path = &sp
+		}
+
 		if err := s.DB.Create(&CollectionRef{
 			Collection: col.ID,
 			Content:    content.ID,
+			Path:       path,
 		}).Error; err != nil {
 			return err
 		}
 	}
 
-	return c.JSON(200, createContentResponse{
+	return c.JSON(200, util.ContentCreateResponse{
 		ID: content.ID,
 	})
 }
```

Comment:
What happens if `req.Collection != ""` (aka a collection uuid is present) but `req.CollectionPath == ""` (aka no or empty collection path provided)? I _think_ we'll need a default path (`/filename`) to be set.

Reply:
since `path` is a pointer, it's initial value would be `nil`. since `req.CollectionPath == ""`, the pointer never gets updated, and in the call to `DB.create`, the value of the `Path` db column will also be nil, leaving it in it's default state (which should be what is expected here)

Reply:
Makes sense! I think we definitely need to revisit this whole path logic some time in the future though.